README.md

```diff
@@ -85,6 +86,19 @@ pip install dohq-artifactory==0.5.dev243
 ```
 
 # Usage
+## Enable logging 
+For purpose of debugging or need of additional information you can configure logger to work with current library
+Just configure `logging` module in your python script. Simpliest example:
+~~~python
+import logging
+from artifactory import ArtifactoryPath
+
+logging.basicConfig(level=logging.DEBUG)
+path = ArtifactoryPath(
+    "http://my-artifactory/artifactory/myrepo/restricted-path", apikey="MY_API_KEY"
+)
+~~~
+This will add debug messages to StdOut
```

Comment:
Too obvious message, we can remove it

Reply:
Or let's move it above. Descriptions should be before examples. 

Could we use `console` instead of `StdOut`?

Reply:
done